### PR TITLE
chore:update vitepress

### PR DESCRIPTION
**Pull request by shaonianche**
Signed-off-by: shaoanianche <github@duanfei.org>

update vitepress version.

replace tabs with [code-groups](https://vitepress.vuejs.org/guide/markdown.html#code-groups)

### CODE REVIEW

**Comment by gaocegege** [comment]
Could you also update the docs-zh?

**Comment by shaonianche** [comment]
> Could you also update the docs-zh?

sure ~

**Comment by gaocegege** [comment]
Thanks! Please ping me if it is ready.

**Comment by shaonianche** [comment]
> Thanks! Please ping me if it is ready.

Duplicate with #227 , it's a little more complete

**Comment by gaocegege** [comment]
/cc @VoVAllen 

Should we merge this first?

File: docs-zh/.vitepress/cache/deps/_metadata.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "hash": "4a411804",
+  "browserHash": "a311c120",
+  "optimized": {
+    "vue": {
+      "src": "../../../node_modules/.pnpm/vue@3.2.39/node_modules/vue/dist/vue.runtime.esm-bundler.js",
```
Comment (VoVAllen) [inline docs-zh/.vitepress/cache/deps/_metadata.json:6]: Should those files under `.vitepress/cache` be included in the repo?

**Comment by VoVAllen** [comment]
Other files look good to me. We can merge this first!

File: docs-zh/.vitepress/cache/deps/_metadata.json (hunk 1 above)
Comment (VoVAllen) [reply docs-zh/.vitepress/cache/deps/_metadata.json:6]: https://github.com/vuejs/vitepress/pull/1700/files
Seems it should be excluded

**Pull request merged**